python/pyspark/ml/util.py

```diff
@@ -237,6 +300,13 @@ def _load_java_obj(cls, clazz):
             java_obj = getattr(java_obj, name)
         return java_obj
 
+    @classmethod
+    def _load_given_name(cls, java_class):
```

Comment:
Please also stick with camelCase for methods since that's what pyspark does.  (here and elsewhere)